pkg/report/writer.go

```diff
@@ -50,28 +50,28 @@ func WriteResults(format string, output io.Writer, results Results, outputTempla
 		return xerrors.Errorf("unknown format: %v", format)
 	}
 
-	if err := writer.Write(results); err != nil {
+	if err := writer.Write(results, severities); err != nil {
 		return xerrors.Errorf("failed to write results: %w", err)
 	}
 	return nil
 }
 
 type Writer interface {
-	Write(Results) error
+	Write(Results, []dbTypes.Severity) error
 }
 
 type TableWriter struct {
 	Output io.Writer
 	Light  bool
 }
 
-func (tw TableWriter) Write(results Results) error {
+func (tw TableWriter) Write(results Results, severities []dbTypes.Severity) error {
```

Comment:
How about adding `severities` as the field of `TableWriter`? This is because this parameter is needed only for `TableWriter`.

```
type TableWriter struct {
	Output io.Writer
	Light  bool
	severities  []dbTypes.Severity
}
```

We probably don't need to export this field. I exported `Output` and `Light` before, but it doesn't look necessary now 😄